partner_relations/models/res_partner_relation_type_selection.py

```diff
@@ -1,149 +1,128 @@
 # -*- coding: utf-8 -*-
 # © 2014-2016 Therp BV <http://therp.nl>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-'''
-Created on 23 may 2014
-
-@author: Ronald Portier, Therp
-
-rportier@therp.nl
-http://www.therp.nl
-
+"""
 For the model defined here _auto is set to False to prevent creating a
-database file. All i/o operations are overridden to use a sql SELECT that
-takes data from res_partner_connection_type where each type is included in the
+database file. The model is based on a SQL view based on
+res_partner_relation_type where each type is included in the
 result set twice, so it appears that the connection type and the inverse
 type are separate records..
 
 The original function _auto_init is still called because this function
 normally (if _auto == True) not only creates the db tables, but it also takes
 care of registering all fields in ir_model_fields. This is needed to make
 the field labels translatable.
-
-example content for last lines of _statement:
-select id, record_type,
-  customer_id, customer_name, customer_city, customer_zip, customer_street,
-  caller_id, caller_name, caller_phone, caller_fax, caller_email
-from FULL_LIST as ResPartnerRelationTypeSelection where record_type = 'c'
-ORDER BY ResPartnerRelationTypeSelection.customer_name asc,
-ResPartnerRelationTypeSelection.caller_name asc;
-
-'''
+"""
 from psycopg2.extensions import AsIs
+
 from openerp import api, fields, models
 from openerp.tools import drop_view_if_exists
+
 from .res_partner_relation_type import ResPartnerRelationType
-from .res_partner import PADDING
+
+
+PADDING = 10
```

Comment:
yes, thanks for elaborating, they have nothing to do with each other. A quick glance makes me think that there are also no instance where we'd need a PADDING1 and PADDING2 to differenciate between both, so let's keep it the way it is. The other point was indeed quite theoretical and only meant for the case you touch this anyways. Then let's figure out my two general questions and then I'd merge immediately